Sources/Datadog/RUM/Instrumentation/RUMCommandSubscriber.swift

```diff
@@ -27,3 +27,10 @@ internal protocol RUMCommandPublisher: AnyObject {
     /// - Parameter subscriber: The RUM command subscriber.
     func publish(to subscriber: RUMCommandSubscriber)
 }
+
+internal final class GlobalRUMCommandSubscriber: RUMCommandSubscriber {
+    func process(command: RUMCommand) {
+        let subscriber = Global.rum as? RUMCommandSubscriber
+        subscriber?.process(command: command)
+    }
+}
```

Comment:
I'm not a fan of this, but it helps for injecting a `RUMCommandSubscriber` instance for testing. Ultimately, The `Global.rum` and this class will be removed.

Reply:
Maybe worth documenting the usage and plan in the header comment?